diff --git a/include/fastdds/rtps/transport/TransportDescriptorInterface.h b/include/fastdds/rtps/transport/TransportDescriptorInterface.h
index 149707736..2b880cc18 100644
--- a/include/fastdds/rtps/transport/TransportDescriptorInterface.h
+++ b/include/fastdds/rtps/transport/TransportDescriptorInterface.h
@@ -17,9 +17,7 @@
 
 #include <fastrtps/fastrtps_dll.h>
 
-#ifdef _WIN32
 #include <cstdint>
-#endif // ifdef _WIN32
 #include <vector>
 #include <string>
 
diff --git a/include/fastrtps/types/TypeNamesGenerator.h b/include/fastrtps/types/TypeNamesGenerator.h
index a8f45e013..5554eeaa9 100644
--- a/include/fastrtps/types/TypeNamesGenerator.h
+++ b/include/fastrtps/types/TypeNamesGenerator.h
@@ -3,6 +3,7 @@
 
 #include <fastrtps/fastrtps_dll.h>
 
+#include <cstdint>
 #include <string>
 #include <vector>
 
